jenkinsfiles/ginkgo-kubernetes-all.Jenkinsfile

```diff
@@ -129,7 +129,7 @@ pipeline {
                 }
             }
         }
-        stage('Copy code and boot VMs 1.{14,15}'){
+        stage('Copy code and boot VMs 1.{17,18}'){
```

Comment:
@nbusseneau I think it's pointless to change this file, should we delete it on master?

Reply:
Yes, it is not used anymore on `master`, can be removed. We kept it while transitioning quarantine builds but this has been done since then.